connectors/slack/template/app.js

```diff
@@ -0,0 +1,180 @@
+const Express = require('express');
+const { createEventAdapter } = require('@slack/events-api');
+const Superagent = require('superagent');
+const Sdk = require('@fusebit/add-on-sdk');
+const storage = require('./storage');
+
+const app = Express();
+
+var slackEvents;
+function ensureEventAdapter(ctx){
+
+    if(!slackEvents){
+        slackEvents = createEventAdapter(ctx.configuration.slack_signing_secret,
+        {
+            includeBody: true
+        });
+
+        // Need to listen to all events, so hack in global listener
+        const oldEmit = slackEvents.emit;
+        slackEvents.emit = function(type, event, body, ...args) {
+            onAll(event, body);
```

Comment:
I think the idiom requires checking the return value to see if it's been 'cancelled'?

Reply:
Can you elaborate? I don't see support for this in the way Slack implements the `EventEmitter` either https://github.com/slackapi/node-slack-sdk/blob/main/packages/events-api/src/http-handler.ts

Reply:
We bypassed this now and are not using Slack's emmitter